onnxruntime/core/providers/cuda/cuda_graph.h

```diff
@@ -3,32 +3,60 @@
 
 #pragma once
 
+#include <unordered_map>
+
 #include "core/common/common.h"
 #include "core/platform/ort_mutex.h"
 #include "core/providers/cuda/cuda_pch.h"
 
 namespace onnxruntime {
 
-using CaptureId_t = unsigned long long;
+using CudaGraphAnnotation_t = int;
+using CudaGraphSet_t = std::unordered_map<CudaGraphAnnotation_t, cudaGraphExec_t>;
```

Comment:
I thought that is better to design as std::unordered_map<CudaGraphAnnotation_t, **CudaGraph**>

Reply:
Do we need to store CudaGraph_t? since the type cudaGraphExec_t is the one that being replayed

Reply:
It is odd to see that CudaGraph contains a member of CudaGraphSet.

Reply:
ah I see what you mean